src/ShopifyApp/ShopifyAppProvider.php

```diff
@@ -103,12 +103,7 @@ public function register()
 
             // Queriers
             IShopQuery::class => [self::CSINGLETON, function () {
-                $model = $this->app['config']->get('auth.providers.users.model');
-                $modelInstance = new $model();
-
-                return new ShopQuery(
-                    $modelInstance
-                );
```

Comment:
Ah! Didn't realize I still had this left in, since its internally handled in the class now.